### ISSUE TITLE
Image not found if in subfolder with space in | media

### ISSUE THREAD

**Issue by Zmove** [abwebdevelopers/oc-imageresize-plugin#30]
To reproduce :

Create a folder with a space in media directory, for example `my folder/dummy.jpg`

In your template, try to resize the image : `<img src="{{ 'my folder/dummy.jpg'|media|resize(500) }}" />`

You will get the image not found image instead of the one you defined.

If you replace "my folder" with "my_folder", it works, the problem come from the space in the folder name.

But it needs to be handled by this module, you can't explain to non technical client that they can't use space in their folder name (whereas the media library alow it).

Best regards,

**Comment by LukeTowers** [comment]
@bennothommo can you add a test for this in the October core implementation?

**Comment by JumpLink** [comment]
Since the last update we also have the problem

**Comment by daslicht** [comment]
It doen't even work under some circumstances (still unclear which) if the name contains no spaces

**Comment by LukeTowers** [comment]
@Zmove @JumpLink @daslicht if you guys could test out https://github.com/octobercms/october/pull/5231 that would be great

**Comment by JumpLink** [comment]
@LukeTowers Are you working on core support for the feature? That`s great!

**Comment by LukeTowers** [comment]
@JumpLink yes, it's been merged and will be available as a part of the L6 update in Build 469.
